src/pages/_app.js

```diff
@@ -117,7 +135,13 @@ export default class App extends BaseApp {
       }
     }
 
-    const head = routeName ? getHead(routeName, props) : null;
+    const { name: productName, description } = clientSideConfiguration.textContent.product;
+    const shareHost = `https://${hostName}/`;
+
+    // TODO Find a cleaner way to check if this is the main wheelmap app …
+    // Might not be needed anymore once the clientSideConfiguration contains information for
+    // Twitter and Google Analytics
+    const isMainApp = productName === 'Wheelmap';
```

Comment:
0: Ah, now I see. Yes - this stuff should be part of the clientSideConfiguration at some point.